hydra_genetics/commands/create.py

```diff
@@ -392,7 +392,7 @@ def extract_run_information(file_path, number_of_reads=200, every_n_reads=1000,
     :type number_of_reads: integer
     :param: warning_threshold: raise a warning for char with lower occurences this value
     :type warning_threshold: float
-    :param compare_first_and_last_read: compare first read with last read to detect merged lanes or runs
+    :param compare_first_and_last_read: compare first read with last read to detect merged lanes or s
```

Comment:
What does `s` stand for?

Reply:
I did a replace one "run" and must have accidentally replaced it with a empty string instead of flowcell

Reply:
fixed now